src/olympia/amo/utils.py

```diff
@@ -642,7 +641,21 @@ def resize_image(source, destination, size=None, *, format='png', quality=80):
             im.load()
     original_size = im.size
     if size:
-        im = processors.scale_and_crop(im.convert('RGBA'), size)
+        try:
+            target_width, target_height = [int(v) for v in size]
+            source_width, source_height = [float(v) for v in im.size]
```

Comment:
You don't need to `int()` the `size` values - they're from a constant.  
`float()` on the im.size values seems unnecessary also as python automatically returns a float from a `/` operation even if both numbers are integers.

Reply:
This will be enough to expand
```suggestion
            target_width, target_height = size
            source_width, source_height = im.size
```